README.md

```diff
@@ -70,8 +70,33 @@ isValidPhoneNumber(invalidPhoneNumber); // Returns `false` - "311" is not a vali
 const intlNumber = '+380 97 123 4567';
 isValidPhoneNumber(intlNumber); // Returns `true` - "380" is the region code for Ukraine
 
-const invalidIntlNumber = '+666 97 123 4567';
-isValidPhoneNumber(invalidIntlNumber); // Returns `false` - "666" is not a valid region code
+const invalidIntlNumber = '+000 97 123 4567';
```

Comment:
Changed from `666` region code since the regex may confuse this with Thailand: https://github.com/hearsaycorp/phonenumber-util/blob/main/src/phoneCodes.js#L834